### PR TITLE
Update helm-install.md

### PR DESCRIPTION
**Pull request by veggiemonk**
Related to:
* https://github.com/istio/istio.github.io/pull/1533
* https://github.com/istio/old_issues_repo/issues/387
* https://github.com/istio/istio/pull/6348

### CODE REVIEW

File: content/docs/setup/kubernetes/helm-install.md [hunk 1]
```diff
@@ -32,13 +32,13 @@ recommended.
       (requires Kubernetes >=1.9.0):
 
         ```command
-        $ helm template @install/kubernetes/helm/istio@ --name istio --namespace istio-system > $HOME/istio.yaml
+        $ helm template @install/kubernetes/helm/istio@ --name istio --namespace istio-system --set global.tag=0.8.0 > $HOME/istio.yaml
```
Comment (gyliu513) [inline content/docs/setup/kubernetes/helm-install.md]: @geeknoid should this be shipped with 0.8?

Comment (geeknoid) [reply content/docs/setup/kubernetes/helm-install.md]: Are you asking whether to include this doc change in the release-0.8 branch so it goes to istio.io right away?

If so, then it probably makes sense to merge this change into release-0.8.

Comment (gyliu513) [reply content/docs/setup/kubernetes/helm-install.md]: @geeknoid I think this should only goes to 0.8 release but not master. In master, we should probably use the latest branch.

Comment (veggiemonk) [reply content/docs/setup/kubernetes/helm-install.md]: created a new PR https://github.com/istio/istio.github.io/pull/1538

Comment (sdake) [reply content/docs/setup/kubernetes/helm-install.md]: The default tag in 0.8.0 values.yaml file is 0.8.0 so I'm not sure this makes sense against 0.8.0 branch of docs.

There may be other use cases that I don't understand.  Could you expand on the problem your trying to solve?

Thanks
-steve

Comment (gyliu513) [reply content/docs/setup/kubernetes/helm-install.md]: +1 to @sdake , Yes, 0.8 release already using `0.8.0` as default tag, so seems we do not need this PR.

**Comment by veggiemonk** [comment]
@sdake  The main PR was here: https://github.com/istio/istio/pull/6348

The problem was not with the build but with the documentation. Hence this PR.

The discussion is a bit split in between PR now
https://github.com/istio/istio.github.io/pull/1540#issuecomment-398292066

**Comment by geeknoid** [comment]
Hi, I did some refactoring and the file you edited has moved in the repo. Please rebase and push again. Sorry for the inconvenience.

**Comment by veggiemonk** [comment]
@geeknoid no worries. Rebasing is done

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit c75789e9a9d149b729af06f1e1e52fed07430ac4

https://app.netlify.com/sites/preliminary-istio/deploys/5b33b8a667610c13f1a3cccb

**Comment by veggiemonk** [comment]
@gyliu513 I have no problem with closing this PR. But again, following the document "most users" won't use the Makefile. 
It's either documented or having sane defaults

**Comment by gyliu513** [comment]
@veggiemonk  For master branch, most of the users are developers ;-)

**Pull request closed without merging**